servicetalk-http-router-jersey/src/main/java/io/servicetalk/http/router/jersey/CharSequenceUtils.java

```diff
@@ -26,23 +26,4 @@ private CharSequenceUtils() {
     static CharSequence asCharSequence(final Object o) {
```

Comment:
This method is used in a single place, can you please move it there and kill this class?